lib/matplotlib/_api/__init__.py

```diff
@@ -297,3 +298,20 @@ def warn_external(message, category=None):
             break
         frame = frame.f_back
     warnings.warn(message, category, stacklevel)
+
+
+def g_sig_digits(value, delta):
```

Comment:
This function is not API related. I think it should be `cbook._g_sig_digits`.

Reply:
Sure.